app/routes.py

```
@@ -0,0 +1,587 @@
+        </div>
+        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"></script>
+        <script src="https://stackpath.bootstrap.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    </body>
+    </html>
+    '''
+    return render_template_string(template)
+
+
+
+from flask import Blueprint, render_template, request, flash, redirect, url_for
```

Ruff (0.8.0)

Redefinition of unused `render_template` from line 203

Remove definition: `render_template`

(F811)